app/Models/customer.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace App\Models;
+
+use App\DB\Model;
+use App\Core\DB;
+use PDO;
+use Exception;
+
+class Customer extends Model {
+    private ?int $id;
+    private ?string $name;
+    private ?string $email;
+    private ?string $address;
+
+    public function __construct(PDO $pdo, ?int $id = null) {

```

Comment:
`$pdo` connection-in qiragdan gelmesine ehtiyac yoxdur, `DB:DB()`-dan access etmek olur